frontend/src/shared/helpers.tsx

```diff
@@ -78,19 +82,40 @@ export const requestColDefs: ColDef[] = [
     headerName: "View Samples",
     cellRenderer: (params: ICellRendererParams) => {
       return (
-        <Button
-          variant="outline-secondary"
-          size="sm"
-          onClick={() => {
-            if (params.data.igoRequestId !== undefined) {
-              params.context.navigateFunction(
-                `/requests/${params.data.igoRequestId}`
-              );
-            }
-          }}
-        >
-          View
-        </Button>
+        <div style={{ display: "flex", alignItems: "center", height: "100%" }}>
+          <Button
+            variant="outline-secondary"
+            size="sm"
+            onClick={() => {
+              if (params.data.igoRequestId !== undefined) {
+                params.context.navigateFunction(
+                  `/requests/${params.data.igoRequestId}`
+                );
+              }
+            }}
+          >
+            View
+          </Button>
+        </div>
+      );
+    },
+    sortable: false,
+  },
+  {
+    headerName: "Status",
+    cellRenderer: (params: ICellRendererParams<DashboardRequest>) => {
+      if (!params.data) return null;
+      const { igoRequestId, validationStatus, validationReport } = params.data;
+      return (validationStatus === false || validationStatus === null) &&
+        validationReport !== '{"samples":[]}' ? (
```

Comment:
Some requests had `validationStatus == false` and `validationReport == '{"samples":[]}'`. This report was vague and I wasn't sure how to handle it. Thoughts? 

Out of a few requests I found with this validation report, they all had >= 1 valid samples, so it doesn't seem like a "all request samples failed validation" situation.

Query to get examples:
```neo4j
MATCH (rm:RequestMetadata)-[:HAS_STATUS]->(st:Status {validationStatus: false})
WHERE st.validationReport CONTAINS ":[]}"
RETURN rm, st limit 5
```

Reply:
The requests with `validationStatus == false` and `validationReport == '{"samples":[]}'` likely imported when stricture validation rules were in place (before we started allowing the flexibility of importing without any valid samples).

Reply:
Okay so data is fixed now and I'm making a PR to the request filter to fix the request-level validation report for these cases

Reply:
Thank you for clarifying and fixing this data!